app/controllers/admin/books_controller.rb

```diff
@@ -26,7 +27,25 @@ def create
     end
   end
 
+  def destroy
+    respond_to do |format|
+      if @book&.destroy
```

Comment:
Vì đã dùng filter check book tồn tại trc đó rồi nên ở đây ko cần check nữa
```suggestion
      if @book.destroy
```